src/debug/DebugAssetsInitializer.ts

```diff
@@ -146,6 +147,10 @@ export class DebugAssetsInitializer {
             connectPreLaunchTask[`isolateAs`] = isolateAs;
         }
 
+        if (containerName != null) {
```

Comment:
💡 bit: can this be `undefined` or `empty` ? might be do length check?

Reply:
Changed this.